client/src/components/Lists/Transactions.js

```diff
@@ -135,10 +152,17 @@ export class Transactions extends Component {
 			}, 60000);
 			this.searchTransactionList(nextProps.currentChannel);
 		}
+		if(nextProps.transactionListSearchQuery!==this.props.transactionListSearchQuery){
+			this.setState({query:nextProps.transactionListSearchQuery});
+		}
+	}
+	componentDidUpdate(prevProps,prevState) {
+		if(prevState.page!=this.state.page || prevState.rowsPerPage!=this.state.rowsPerPage || this.state.searchClick){
+			this.handleSearch();
+		}
 	}
-
 	componentWillUnmount() {
-		clearInterval(this.interVal);
+		clearInterval(this.interval);
 		if (this.props.transactionId) {
```

Comment:
resolved - interval not getting cleared on unmount in existing code. Please review the latest commits @adityajoshi12